gtfs/models/base.py

```diff
@@ -14,6 +14,18 @@ class Meta:
         abstract = True
 
 
+class PriceModel(models.Model):
+    price = models.DecimalField(
+        verbose_name=_("price"), max_digits=10, decimal_places=2
+    )
+    currency_type = models.CharField(
+        max_length=3, help_text=_("ISO 4217 alphabetical currency code")
```

Comment:
Missing `verbose_name`. Someone might blame the original author of that field.

Reply:
Fixed. :wink: